components/app/footer/PrimaryFooter.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <div class="bg-base-200">
+    <section-container>
+      <footer
```

Comment:
suggestion: Using 2 footer tags on the same level feels wrong. Would wrap both sections in 1 tag.

Reply:
You are right, two footer elements shouldn't even be in the same section. Implemented a solution with one footer element as wrapper for the whole component.